### PR TITLE
UI/financial-verification

### PR DESCRIPTION
**Pull request by luisgov**
This PR fixes some layout and text styling bugs. It also replaces the input fields with the TextInput module and adds tooltips.

screencapture-localhost-8000-requests-verify-5de9ff38-eb3b-4acf-b45d-0bc2c0ead6ed-2018-08-14-15_21_20

### CODE REVIEW

File: templates/requests/financial_verification.html [hunk 1]
```diff
@@ -20,192 +24,47 @@ <h2 id="financial-verification">Financial Verification</h2>
...
-        {{ f.ba_code(placeholder="Example: 02") }}
-        {% for e in f.ba_code.errors %}
-          <div class="usa-input-error-message">
-            {{ e }}
-          </div>
-        {% endfor %}
-
-        <!-- KO Information -->
+        {{ TextInput(f.task_order_id,placeholder="Example: 1234567899C0001",tooltip="Note that there may be a lag between the time you have created and approved the task order to the time it is searchable within the electronic. <br>A Contracting Officer will likely be the best source for this number.") }}
+        {{ TextInput(f.uii_ids,placeholder="Example: \nDI 0CVA5786950 \nUN1945326361234786950",tooltip="A Unique Item Identifer is a unique code that helps the Department of Defense track and report on where and how digital assets are stored. <br>Not all applications have an existing UII number assigned.") }}
```
Comment (dandds) [inline templates/requests/financial_verification.html]: This isn't a text area anymore? The placeholder text reads as one UII now, where previously there were linebreaks.

File: templates/requests/financial_verification.html [hunk 2]
```diff
@@ -20,192 +24,47 @@ <h2 id="financial-verification">Financial Verification</h2>
...
+        {{ TextInput(f.pe_id,placeholder="Example: 0203752A",tooltip="Program Element numbers helps the Department of Defense identify which offices\\' budgets are contributing towards this resource use.") }}
+        {{ TextInput(f.treasury_code,placeholder="Example: 1200") }}
+        {{ TextInput(f.ba_code,placeholder="Example: 02") }}
 
         <h3>Contracting Officer (KO) Information</h3>
+        {{ TextInput(f.fname_co,placeholder="Contracting Officer First Name") }}
+        {{ TextInput(f.lname_co,placeholder="Contracting Officer Last Name") }}
+        {{ TextInput(f.email_co,placeholder="jane@mail.mil") }}
```
Comment (dandds) [inline templates/requests/financial_verification.html]: Do we want frontend validations on these fields?

File: templates/requests/financial_verification.html [hunk 3]
```diff
@@ -20,192 +24,47 @@ <h2 id="financial-verification">Financial Verification</h2>
...
-        {% endfor %}
+        {{ TextInput(f.fname_cor,placeholder="Contracting Officer Representative First Name") }}
+        {{ TextInput(f.lname_cor,placeholder="Contracting Officer Representative Last Name") }}
+        {{ TextInput(f.email_cor,placeholder="jane@mail.mil") }}
+        {{ TextInput(f.office_cor,placeholder="Example: WHS") }}
 
         <br><hr>
         <em>&darr; FIELDS NEEDED FOR MANUAL ENTRY OF TASK ORDER INFORMATION (only necessary if EDA info not available)</em>
 
+        {{ TextInput(f.funding_type) }}
```
Comment (dandds) [inline templates/requests/financial_verification.html]: This is a still a `SelectField` in the wtform object. Is it supposed to be a regular text input now?

**Comment by luisgov** [comment]
@dandds I updated the fields and added some validation. I will have to add more robust validation on some other fields based on the requirements on the spreadsheet, will have to tackle that at another time.

**Pull request merged**